api/tests/Api/ContentNodes/UpdateContentNodeTestCase.php

```diff
@@ -51,4 +51,39 @@ public function testPatchIsAllowedForManager() {
         $this->patch(user: static::$fixtures['user1manager']);
         $this->assertResponseStatusCodeSame(200);
     }
+
+    /**
+     * @dataProvider getContentNodesWhichCannotHaveChildren
+     */
+    public function testCreateValidatesThatParentCanContainChildren(string $idOfParentFixture) {
```

Comment:
```suggestion
    public function testCreateValidatesThatParentCannotContainChildren(string $idOfParentFixture) {
```

Reply:
I would formulate it the positive way

Reply:
Postitive or negative wording... I have no positive/negative feelings against each, but the description is still wrong 😀 

This translates to: "Überprüfen, **dass** der Parent Children enthalten **darf**"

This is not what you're doing here. You're validating that the parent(s) cannot (=are not allowed) to contain children.

Reply:
done